lib/dialects/postgres/query-interface.js

```diff
@@ -154,4 +154,21 @@ function ensureEnums(qi, tableName, attributes, options, model) {
 }
 
 
+/**
+ A wrapper that fixes MySQL's inability to cleanly remove columns from existing tables if they have a foreign key constraint.
+
+ @param  {QueryInterface} qi
+ @param  {string} tableName     The name of the table.
+ @param  {string} columnName    The name of the attribute that we want to remove.
+ @param  {object} options
+
+ @private
+ */
+function removeColumn(qi, tableName, columnName, options) {
+  options = options || {};
```

Comment:
you can use a default parameter here:
```js
function removeColumn(qi, tableName, columnName, options = {}) {
```

Reply:
This snip is good, but there is a small problem, if executed passing null, for example, imagine this following function invocation:

```js
removeColumn(qi, tableName, columnName, null)
```

The options passed will be null instead of an empty object... Actually I'm not too sure if this could trigger an error, but, if it is required to assert it, it'll be necessary to do something close to what he did.

But also it isn't good practice to modify function attributes, I think you could create a new variable instead of modifying the `options`

Reply:
Please use the default parameter